### ISSUE TITLE
[Bug]: Outputting armour to AP value in chat

### ISSUE THREAD

**Issue by MattZeeX** [raethkcj/RatingBuster#119]
### Describe the bug

![image](https://user-images.githubusercontent.com/49168219/201297948-0376a7f2-7ba7-4588-83e6-5cd4def4a2ce.png)

Ok so I don't *know* that this bug is attributed to RatingBuster, but I am quite certain because the format of the text printed to chat is the same as what is present in the RatingBuster tooltip, and also I can't possibly fathom any other addon or WeakAura handling the bonus Attack Power gained from Armour. Additionally, I think the only thing I changed since yesterday was updating RatingBuster.

So all that being said, whenever I mouse over an item with armor in the tooltip, RatingBuster is printing the value of the Attack Power given into the chat. It gets kinda spammy is kinda annoying haha, I'm sure there's just a debug print somewhere so when you have the time, please remove it haha.

Thank you!

### Addon Source

Curseforge

### Version

3.4.0.30

### Locale

English

### Talent Specialization

Frost Death Knight

### Race

Draenei

**Comment by MattZeeX** [comment]
Ok sorry I've posted this in the wrong github, I am really confused now. I've downloaded my RatingBuster from Curseforge, and the "source" on Curseforge links me to this Github https://github.com/CKNOEP/RatingBuster. In my addon folder's version of RatingBuster, I found the issue [here](https://github.com/CKNOEP/RatingBuster/blob/main/RatingBuster.lua#L3523), but I noticed that when I went to look at your file, I couldn't find the issue at that line number, and also your RatingBuster.lua is completely different than the one I had installed.

So it seems like there are two RatingBusters? Yours are wowinterface and stuff, and the one I use is from Curseforge. But in the bug report template I used above, it gave me the option to say I got the addon from Curseforge, so do you not have the addon on Curseforge? I guess, why are there two version of RatingBuster? I'll close this issue now because it's not really your issue, but I'm just really confused.

**Comment by raethkcj** [comment]
Hi, thanks for the report and feedback. I agree it's an incredibly confusing situation, and there have been several report mix-ups before now.

I began this port of RatingBuster in January 2021 in preparation for TBC Classic, and maintained it through all of TBC. CKNOEP separately began their own port in, I believe, August 2022. I found out about that version for the first time when someone reported a bug from it here. I posted an issue (https://github.com/CKNOEP/RatingBuster/issues/5) to make sure CKNOEP knew this version existed, but they seem to want to keep working on their own.

The reason Curseforge exists as a dropdown option here is exactly so I can tell when a bug report is for the other version of the addon. If I could, I would make it give an error message to redirect you to the proper repository when you try to select Curseforge in the form, but Github's issue forms are pretty new and limited.

As for why my version was never published to Curseforge, I posted about it here: https://github.com/raethkcj/RatingBuster/issues/63

Curseforge claims they want what's best for the WoW community and addon developers, and they even claim that they "welcome competition", but they explicitly refuse to participate in an open platform like Wowup, which was ready and willing to be compatible with both Curseforge and Wago's ecosystems.

**Comment by MattZeeX** [comment]
Fair enough. Sorry to have troubled you. I'll give your version a try, but it is pretty frustrating for me to use multiple different addon managers at once. I just use the Curseforge client without Overwolf, but I still have to manually check for updates and use Elvui which is annoying. I'm sure that is what Curse is counting on. I used Curse since the original days, and it was always perfectly adequate. Sometime around Shadowlands launch, I did swap to WowUp and found it to be a satisfactory experience, but Curse pulling the plug on the API meant that WowUp no longer had all the addons I wanted to use, as obviously not every Addon author is willing to provide the integration needed since some of them really rely on the ad revenue from Curse. It's a poor situation to be in and the onus is on Curse.
